src/terminal/terminal_handlers.c

```diff
@@ -1373,3 +1377,22 @@ int guac_terminal_ctrl_func(guac_terminal* term, unsigned char c) {
 
 }
 
+/**
+ * xterm does not implement APC functions and neither do we.
+ */
+int guac_terminal_apc(guac_terminal* term, unsigned char c) {
+
+    /* Look for the "ESC \" (string terminator) sequence,
```

Comment:
Is `ESC \` the only sequence that is valid as the string terminator for APC? Other sequences have involved other characters for the string terminator:

https://github.com/apache/guacamole-server/blob/b61a6ab758177b4d4cec4a3a0a2f51a3e76c3772/src/terminal/terminal_handlers.c#L1335-L1337